mpz_search.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+grep "mpz_[a-z_0-9]*" $1 -a -r -o -h --exclude "mpz_search.sh" | sort -d | uniq
```

Comment:
Functionality-wise, it looks great! 👍 I've only noticed one small thing: currently it searches through all files, which seems unnecessary. Is it possible to grep in only `.c`˙and `.h` files?

Reply:
Thanks!
I'll look for a solution.